laravel/socialite/2015-07-23.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Insecure state generation
+link:      https://github.com/laravel/socialite/pull/91
```

Comment:
it would be good to have a link to something describing the issue in more details. The PR doing the patch does not contain any information (what is the severity of the issue, which cases are affected, ...)